pkg/gui/controllers/switch_to_diff_files_controller.go

```diff
@@ -91,3 +91,11 @@ func (self *SwitchToDiffFilesController) viewFiles(opts SwitchToCommitFilesConte
 
 	return self.c.PushContext(diffFilesContext)
 }
+
+func (self *SwitchToDiffFilesController) itemHasFilesToView(ref types.Ref) *types.DisabledReason {
```

Comment:
I'd add a comment explaining that if we're dealing with an empty commit, this function won't return a disabled reason. It's instead checking whether the selected item could ever have files. Maybe there's a better function name to make that clear and remove the need for a comment but nothing comes to mind

Reply:
I like the idea of choosing function names such that you don't need comments. How about `itemCanHaveFilesToView`? Or even more concrete, `itemHasCommitHash`?

Reply:
Decided on `itemRepresentsCommit`

Reply:
f4492790a561